graph-data.rs/src/check_signatures.rs

```diff
@@ -0,0 +1,149 @@
+use anyhow::Result as Fallible;
+use anyhow::{format_err, Context};
+use futures::stream::{FuturesOrdered, StreamExt};
+use lazy_static::lazy_static;
+use reqwest::{Client, ClientBuilder};
+use semver::Version;
+use std::collections::HashSet;
+use std::ops::Range;
+use std::str::FromStr;
+use std::time::Duration;
+use url::Url;
+
+use cincinnati::plugins::prelude_plugin_impl::TryFutureExt;
+use cincinnati::Release;
+// base url for signature storage - see https://github.com/openshift/cluster-update-keys/blob/master/stores/store-openshift-official-release-mirror
+lazy_static! {
+  static ref BASE_URL: Url =
+    Url::parse("https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/")
+      .expect("could not parse url");
+}
+
+// Signature file request timeout
+static DEFAULT_TIMEOUT_SECS: u64 = 30;
+
+// CVO has maxSignatureSearch = 10 in pkg/verify/verify.go
+static MAX_SIGNATURES: u64 = 10;
+
+// Skip some versions from 4.0 / 4.1 / 4.2 times
+// https://issues.redhat.com/browse/ART-2397
+static SKIP_VERSIONS: &[&str] = &[
+  "4.0.0-0.9+amd64",
+  "4.0.0-0.10+amd64",
+  "4.0.0-0.11+amd64",
```

Comment:
no need to skip these since #528 , right?

Reply:
Ah, correct. Removed those, lets see